src/Visualization/Python/GenerateTetrahedralConnectivity.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+from typing import Optional
+
+import click
+import h5py
+import numpy as np
+import rich
+import scipy.spatial as spatial
+
+from spectre.support.CliExceptions import RequiredChoiceError
+from spectre.support.Logging import configure_logging
+from spectre.Visualization.GenerateXdmf import available_subfiles
+
+logger = logging.getLogger(__name__)
+
+
+def generate_tetrahedral_connectivity(
+    h5files,
+    subfile_name: str,
+    relative_paths: bool = True,
+    start_time: Optional[float] = None,
+    stop_time: Optional[float] = None,
+    stride: int = 1,
+    coordinates: str = "InertialCoordinates",
+    delete_tetrahedral_connectivity: bool = False,
+):
+    """Generate tetrahedral connectivity using scipy.spatial.Delaunay
+
+    Given the coordinates, this generates a tetrahedral connectivity that can
+    be read in by ParaView or other visualization software (e.g. VisIt or yt).
+    It uses the scipy.spatial.Delaunay class to generate the connectivity, which
+    uses the Qhull library (github.com/qhull/qhull/), which uses the quickhull
+    algorithm and scales as O(n^2) in the worst case. Thus, generating the
+    connectivity can take several minutes per temporal ID. Unfortunately,
+    depending on the particular grid point distribution, generating the
+    connectivity may even fail in qhull, which is unfortunately difficult
+    to fix.
```

Comment:
Add a note that this is faster than ParaView's Delaunay filter and only has to be done once, since it's stored in the file